server/api/db/sqldb/models.py

```diff
@@ -769,6 +769,36 @@ class TimeWindowTracker(Base, mlrun.utils.db.BaseModel):
         def get_identifier_string(self) -> str:
             return f"{self.key}"
 
+    class AlertHistory(Base, mlrun.utils.db.BaseModel):
+        __tablename__ = "alert_history"
+        __table_args__ = (
+            PrimaryKeyConstraint("activation_time", "id", name="_alert_history_uc"),
+        )
```

Comment:
I suggest to add indexes according to the expected lists we will want to support (from ui perspective) 
like "all project alerts" -> add index on projects / "all kind X / entity" and so on. I suggest to follow the UI narrative to begin with